build/gocommand.js

```diff
@@ -17,38 +17,120 @@
  */
 import child from 'child_process';
 import lodash from 'lodash';
+import Q from 'q';
 
 import conf from './conf';
+let env = null;
 
 /**
- * Spawns Go process wrapped with the Godep command. Backend source files must me packaged with
- * 'package-backend-source' task before running this command.
+ * Spawns a Go process wrapped with the Godep command after making sure all GO prerequisites are
+ * present. Backend source files must be packaged with 'package-backend-source' task before running
+ * this command.
  *
- * @param {!Array<string>} args
- * @param {function(?Error=)} doneFn
+ * @param {!Array<string>} args - Arguments of the go command.
+ * @param {function(?Error=)} doneFn - Callback.
  */
 export default function spawnGoProcess(args, doneFn) {
   // Add base directory to the gopath so that local imports work.
+  let sourceGopath = `${process.env.GOPATH}:${
+      conf.paths.backendTmp}`;
+  let devPath = `${process.env.PATH}:${
+      conf.paths.goTools}/bin`;
+  env = lodash.merge(process.env, {GOPATH: sourceGopath, PATH: devPath});
```

Comment:
Hmm... I see something wrong. You're modifying global variable here. Why dont you do this on global scope and make the var constant ?